src/Auth_Command.php

```diff
@@ -126,6 +140,11 @@ public function create( $args, $assoc_args ) {
 	 * [--admin-tools]
 	 * : Update auth on admin-tools.
 	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Add auth on site and its admin-tools with default username(easyengine) and random password
+	 *     $ ee auth create example.com
```

Comment:
Update the doc-comment for `update` function instead of create.

Add 2-3 examples giving atleast one example for `global` scope as well.

Reply:
Done